docs/panels.rst

```diff
@@ -130,6 +130,11 @@ Profiling information for the processing of the request.
 This panel is included but inactive by default. You can activate it by default
 with the ``DISABLE_PANELS`` configuration option.
 
+The panel will include all function calls made by your project if your using
+the setting ``settings.BASE_DIR`` to point to your project's root directory.
+If a function is in a file within that directory and does not include
+``"/site-packages/"`` in the path, it will be included.
```

Comment:
I had some strange occurrences of broken Python installations when packages ended up in `dist-packages` as well. I couldn't really explain the difference -- all I know is that `site-packages` signify that everything is working well, `dist-packages` signify that many things will end up broken.

Reply:
We could check for that path here too.

Reply:
I think it would be prudent to do so, both  `dist-packages` and `site-packages` are very improbable to exist outside virtualenvs.